### PR TITLE
Extra Directories

### PR DESCRIPTION
**Pull request by amahon**

### CODE REVIEW

**Comment by webpro** [comment]
Hi, @amahon! Apologies for the late reply. Are you still interested in getting this merged? I think it may help some other people as well. E.g. see #143.

File: lib/options.js [hunk 1]
```diff
@@ -80,6 +80,7 @@ const optionList = [
   'scripts',
   'separator',
   'static',
+  'static_dirs',
```
Comment (webpro) [inline lib/options.js]: This should also be`staticDirs` I think (or at least I prefer this for consistency).

File: package.json [hunk 2]
```diff
@@ -47,5 +47,14 @@
...
+  },
+  "bugs": {
+    "url": "https://github.com/amahon/reveal-md/issues"
+  },
+  "homepage": "https://github.com/amahon/reveal-md#readme",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "author": "Andrew Mahon <andrewmahon@fastmail.com> (https://andrewmahon.info)"
```
Comment (webpro) [inline package.json]: Please undo the changes to package.json.

**Comment by Miterion** [comment]
Would love to see this implemented. If @amahon does not react anymore I would gladly make the suggested changes and create a new pull request.

File: lib/options.js (hunk 1 above)
Comment (amahon) [reply lib/options.js]: :+1:

**Comment by amahon** [comment]
After sorting some lurking issues in my local branches, I think I have this all set. I left the package version at `1.0.0` and I cleaned up everything else. Let me know if you'd like any other changes.

Hopefully this can be helpful!

a

**Comment by webpro** [comment]
I've released this (slightly modified) as [v1.1.0](https://github.com/webpro/reveal-md/releases/tag/1.1.0).

**Comment by amahon** [comment]
Awesome! Glad it made it in.
